randovania/games/prime1/exporter/game_exporter.py

```diff
@@ -67,7 +132,7 @@ def export_game(self, patch_data: dict, export_params: GameExportParams,
         try:
             py_randomprime.patch_iso_raw(
                 patch_as_str,
-                py_randomprime.ProgressNotifier(lambda percent, msg: progress_update(msg, percent)),
+                py_randomprime.ProgressNotifier(lambda percent, msg: updaters[2](msg, percent)),
```

Comment:
It should only use `updaters[2]` when actually converting assets, otherwise this will skip the first 66% of the bar.